app/workers/seventh_day_evaluation_worker.rb

```diff
@@ -2,17 +2,20 @@ class SeventhDayEvaluationWorker
   include Sidekiq::Worker
   sidekiq_options queue: 'committee_evaluations'
 
-  def perform(submission_id)
+  def perform(submission_id, final_reminder_sent = false)
     submission = Submission.find(submission_id)
     return unless submission.status_behavior.waiting_for_committee_review?
 
-    approval_status = submission.approval_status_behavior.status
-    if approval_status == 'pending'
+    approval = submission.approval_status_behavior
+    if approval.status == 'pending'
       if current_partner.graduate? && submission.degree_type.slug == 'dissertation'
         dissertation_emails(submission)
       else
         non_dissertation_emails(submission)
       end
```

Comment:
Sorry, I wanted to elaborate further on this when we met, but I forgot.  The Grad School no longer wants emails sent out to the program chair and author here.  They just want a reminder email sent to the committee members that did not vote.  So the logic here is no longer needed, and the `dissertations_emails` is no longer needed.  We can just use `non_dissertations_emails` but the method will have to be renamed.

Reply:
I remember we talked about this briefly, but I thought that was just for if it had 2/3 approval already. So to make sure  I'm clear, even if it's under 2/3, we do not want to send out emails to the program head, chairs, or authors correct?

Reply:
After looking at the issue again, I agree with you.  Let's keep it as it is and send emails to the program head, chairs, and authors when under 2/3 voted.  I think I was confused about when those emails go out.